distribution/docs/src/main/resources/content/_managing/_configuring/configuring-cas.adoc

```diff
@@ -91,8 +91,8 @@ active state. Additionally, it should respond to LDAP requests on the default po
 1389 and 1636.
 . Copy the assigned LDAP keystore and truststore files into the `<DISTRIBUTION>/etc/keystores`
 folder, making sure they overwrite the existing serverKeystore.jks and serverTruststore.jks files.
-. Open the `<DISTRIBUTION>/etc/system.properties file` and make sure the keystore passwords are set
-correctly.
+. Open the `<DISTRIBUTION>/etc/custom.system.properties file` and make sure the keystore passwords
```

Comment:
✏️ @ricklarsen Shouldn't these paths be `${home_directory}/etc` instead of `<DISTRIBUTION>/etc`?

Reply:
yes